teams.json

```diff
@@ -1,5 +1,11 @@
 {
   "teams": [
+    {
```

Comment:
👋  so further down in this doc, there is an `apm` team that includes the stuff I think? the same team manages. Can we rename/use that?

Reply:
do we just want to stick with apm? and make sure the right folks are on it?

Reply:
That would work for me.  Or we can rename it!  I'm fine either way :)